### PR TITLE
Add upgrade instruction for users with outdated ASReview versions

### PR DESCRIPTION
**Pull request by J535D165**
```sh
> asreview lab --no-browser



ASReview LAB version 1.3.2 is available. Please update using:
pip install --upgrade asreview



Serving ASReview LAB at http://localhost:5000/



```

### CODE REVIEW

**Comment by J535D165** [comment]
Don't want an update check? 

```
asreview lab --skip-update-check
```

**Comment by J535D165** [comment]
The update check is not part of development setup.

File: asreview/webapp/entry_points/lab.py [hunk 1]
```diff
@@ -47,21 +50,37 @@ def _deprecated_dev_mode():
...
+
+
+def _check_for_update():
+    """Check if there is an update available."""
+
+    try:
+        r = requests.get("https://pypi.org/pypi/asreview/json")
+        r.raise_for_status()
+        latest_version = r.json()["info"]["version"]
+        if latest_version != __version__ and "+" in __version__:
```
[the remark this answers is not in the document]
Comment (J535D165) [reply asreview/webapp/entry_points/lab.py:72]: Great catch. I accidentally changed this in my last commit while debugging.

Comment (J535D165) [inline asreview/webapp/entry_points/lab.py:72]:
```suggestion
        if latest_version != __version__ and "+" not in __version__:
```

**Review by Rensvandeschoot — APPROVED** [review]

**Pull request merged**